tools/owner-boot-device

```diff
@@ -114,10 +114,10 @@ if which hzn >/dev/null; then
 fi
 
 # Get the other files we need from our git repo, by way of our device binaries tar file
-deviceBinaryDir='sdo_device_binaries_1.8_linux_x64'
+deviceBinaryDir='sdo_device_binaries_1.9_linux_x64'
```

Comment:
Wouldn't this affect 1.8 users as well?

Reply:
No, because the `v1.8` release version of `simulate-mfg.sh` gets `owner-boot-device` from the `v1.8` release also. Download https://github.com/open-horizon/SDO-support/releases/download/v1.8/simulate-mfg.sh and look at that, and you'll see what i mean.